.gitignore

```diff
@@ -100,6 +100,10 @@ Icon?
 ehthumbs.db
 Thumbs.db
 
+# pytest generated files #
+##########################
+/.pytest_cache
```

Comment:
This belongs in its own commit (or even, own PR)

Reply:
Yes: https://github.com/numpy/numpy/pull/10984